python-threatexchange/threatexchange/signal_type/index.py

```diff
@@ -48,6 +48,19 @@ def __init__(self, distance: int, metadata: T) -> None:
         self.metadata = metadata
 
 
+class VPDQIndexMatch(IndexMatch):
+    query_match_percent: float
+    compared_match_percent: float
+
+    def __init__(
+        self, query_match_percent: float, compared_match_percent: float, metadata: T
+    ) -> None:
+        # Dist is meaning less here, because VPDQ match does not return a single dist
+        IndexMatch.__init__(self, -1, metadata)
+        self.query_match_percent = query_match_percent
+        self.compared_match_percent = compared_match_percent
+
+
```

Comment:
blocking: This is a detail of vPDQ, which is not installed by default by the library, and so sohuld live in the vPDQ directory